content/ecosystem/data.toml

```diff
@@ -922,6 +927,7 @@ name = "spirv-reflect"
 source = "crates"
 categories = ["shader"]
 
+
```

Comment:
The `fmt` check is failing due to this extra newline - removing it should fix the build!